aquasec-server/values.yaml

```diff
@@ -0,0 +1,132 @@
+# Specifies the secret data for imagePullSecrets needed to fetch the private docker images
+imageCredentials:
+  repositoryUriPrefix: "docker.io"
+  registry: "index.docker.io/v1/" #for docker.io #REQUIRED
+  username: #REQUIRED
+  password: #REQUIRED
+  email:
+
+rbac:
+  enabled: true
+  roleRef:
+
+db:
+  external:
+    enabled: false
+    name:
+    host:
+    port:
+    user:
+    password:
+    auditName:
+    auditHost:
+    auditPort:
+    auditUser:
+    auditPassword:
+  image:
+    repository: aquasec/database
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+  persistence:
+    enabled: true
+    storageClass:
+    size: 30Gi
+    accessMode: ReadWriteOnce
+  livenessProbe:
+    exec:
+      command:
+      - sh
+      - -c
+      - exec pg_isready --host $POD_IP
+    initialDelaySeconds: 60
+    timeoutSeconds: 5
+    failureThreshold: 6
+  readinessProbe:
+    exec:
+      command:
+      - sh
+      - -c
+      - exec pg_isready --host $POD_IP
+    initialDelaySeconds: 5
+    timeoutSeconds: 3
+    periodSeconds: 5
+  # TODO: Specify the minimal DB resource requirements
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+gate:
+  image:
+    repository: aquasec/gateway
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+    externalPort: 3622
+  replicaCount: 1
+  # TODO: Fill-in the correct liveness and readiness probes
```

Comment:
@oranmoshai can you please spcify the probes for the gate component

Reply:
Currently there is no probe for gateway we will add it to the roadmap